System.IO.Abstractions.TestingHelpers.Tests/MockDirectoryInfoTests.cs

```diff
@@ -254,6 +254,38 @@ public void MockDirectoryInfo_FullName_ShouldReturnNormalizedPath(string directo
             Assert.AreEqual(expectedFullName, actualFullName);
         }
 
+        [TestCase(@"c:\temp\folder  ", @"c:\temp\folder")]
+        [WindowsOnly(WindowsSpecifics.Drives)]
+        public void MockDirectoryInfo_FullName_ShouldReturnPathWithTrimmedTrailingSpaces(string directoryPath, string expectedFullName)
+        {
+            // Arrange
+            directoryPath = XFS.Path(directoryPath);
+            var fileSystem = new MockFileSystem(new Dictionary<string, MockFileData>());
+            var directoryInfo = new MockDirectoryInfo(fileSystem, directoryPath);
+
+            // Act
+            var actualFullName = directoryInfo.FullName;
+
+            // Assert
+            Assert.AreEqual(expectedFullName, actualFullName);
+        }
+
+        [TestCase(@"c:\temp\\folder ", @"folder")]
+        [WindowsOnly(WindowsSpecifics.Drives)]
+        public void MockDirectoryInfo_Name_ShouldReturnNameWithTrimmedTrailingSpaces(string directoryPath, string expectedName)
+        {
+            // Arrange
+            directoryPath = XFS.Path(directoryPath);
```

Comment:
Same as above

Reply:
Thank you. The changes has been committed.